settings.gradle

```diff
@@ -1,2 +1 @@
-// TODO: Update project name.
-rootProject.name = 'template'
+rootProject.name = 'migration'
```

Comment:
Wondering should it be `tis-revalidation-migration`

Reply:
Done